contribs/gmf/src/controllers/abstractmobile.js

```diff
@@ -95,6 +95,13 @@ gmf.AbstractMobileController = function(config, $scope, $injector) {
         config.mapInteractions ||
         ol.interaction.defaults({pinchRotate: false})
   });
+
+  /**
+   * Hide the left navigation when a background layer is selected.
+   */
```

Comment:
I updated the PR so that `mainCtrl.hideNav()` is directly used as discussed:

          gmf-mobile-background-layer-selector-select="mainCtrl.hideNav()">